config/prod/minidream-rstudio-2022-ec2.yaml

```diff
@@ -13,7 +13,7 @@ parameters:
   KeyName: "scicomp"
   VpcName: "computevpc"
   AMIId: "ami-0a3c3e0cb6ae0cc6c"  # packer-base-ubuntu-bionic-v1.0.9
-  InstanceType: "x2gd.8xlarge"     # 32 vCPUs / 512 GiB
+  InstanceType: "r5a.12xlarge"     # 32 vCPUs / 512 GiB
```

Comment:
shouldn't this be `48 vCPUs / 384 GiB`, https://aws.amazon.com/ec2/instance-types/r5/ ?

Reply:
@linglp: And you can update `packer-base-ubuntu-bionic-v1.0.9` to `packer-base-ubuntu-bionic-v1.0.10` in the previous line while you're editing the file.